pkg/virt-handler/vm.go

```diff
@@ -1775,9 +1776,6 @@ func (d *VirtualMachineController) processVmCleanup(vmi *v1.VirtualMachineInstan
 	d.migrationProxy.StopSourceListener(vmiId)
 
 	// Unmount container disks and clean up remaining files
-	if err := d.containerDiskMounter.UnmountKernelArtifacts(vmi); err != nil {
-		return err
-	}
```

Comment:
I'm re-reading this line and I don't understand what I meant originally. Since the code is gone, probably it's better to just disregard this and move along.

Sorry :sweat_smile: